src/Context/SingleVideoContext.js

```diff
@@ -0,0 +1 @@
+// import { createContext, useContext, useReducer } from "react";
```

Comment:
you might re-check this file if you need it or not

Reply:
ok